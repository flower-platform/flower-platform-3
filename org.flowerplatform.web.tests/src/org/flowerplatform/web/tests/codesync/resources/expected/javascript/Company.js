// template BackboneClass
define(function (require) {
    
    "use strict";
    
    var Backbone = require('backbone');// children-insert-point RequireEntry
    
    return Backbone.View.extend({
    
    	render : function (param) {

},
name : John// children-insert-point ClassMember
    
    })

});