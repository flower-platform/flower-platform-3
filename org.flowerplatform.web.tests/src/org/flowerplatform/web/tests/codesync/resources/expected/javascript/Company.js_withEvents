// template BackboneClass
define(function (require) {
    
    "use strict";
    
    var Backbone = require('backbone');// children-insert-point RequireEntry
    
    return Backbone.View.extend({
    
    	render : function (param) {

},
name : John,
events : {
	// children-insert-point EventsAttributeEntry
}// children-insert-point ClassMember
    
    })

});