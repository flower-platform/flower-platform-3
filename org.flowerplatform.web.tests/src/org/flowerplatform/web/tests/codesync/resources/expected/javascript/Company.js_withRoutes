// template BackboneClass
define(function (require) {
    
    "use strict";
    
    var Backbone = require('backbone');// children-insert-point RequireEntry
    
    return Backbone.View.extend({
    
    	render : function (param) {

},
name : John,
routes : {
	// children-insert-point RoutesAttributeEntry
}// children-insert-point ClassMember
    
    })

});