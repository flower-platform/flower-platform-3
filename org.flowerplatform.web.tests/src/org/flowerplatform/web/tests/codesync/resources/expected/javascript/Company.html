<!-- template Form -->
<table class="table table-striped">
	<tr>
	<td>Revenue:</td>
	<td class="view-element"><i class="icon-tag"></i><span><%= revenue %></span> $</td>
	<td><input type="text" class="edit-element" id="revenue" placeholder="<%= revenue %>"/></td>
</tr><!-- children-insert-point FormItem -->
</table>